src/mol-repr/volume/util.ts

```diff
@@ -351,3 +352,46 @@ export function createSegmentTexture2d(volume: Volume, set: number[], bbox: Box3
 
     return textureImage;
 }
+
+/**
+ * Create a new volume that is wrapped by one cell in all dimensions.
+ * Reuses the original volume grid data with new data accessors.
+ */
+export function createWrappedVolume(volume: Volume): Volume {
```

Comment:
Which is the use case of this?
It seems something useful to improve memory usage. Would it be possible to specify an optional new box to apply the volume?